### PR TITLE
Added feature file of no 32

### PR DESCRIPTION
**Pull request by sarangholey**
@DeveshSharma2107 
Please review the Pull Req and allow to merge.

### CODE REVIEW

File: src/test/java/FirstSession/Feature.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package FirstSession;
+
+public class Feature {
+	
+	public static void main(String[] args) {
+		
+		System.out.println("Given");
+		System.out.println("Given");
```
Comment (DeveshSharma2107) [inline src/test/java/FirstSession/Feature.java]: Please change this given to and statement.

File: src/test/java/FirstSession/Feature.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+		System.out.println("When");
+		System.out.println("Then");
+
```
Comment (DeveshSharma2107) [inline src/test/java/FirstSession/Feature.java]: Remove the unused lines.

**Review by DeveshSharma2107 — CHANGES_REQUESTED** [review]

**Comment by sarangholey** [comment]
@DeveshSharma2107 
Made the asked changes please review again.

**Comment by DeveshSharma2107** [comment]
Allowed to merge, please go ahead.

**Pull request merged**